integration-tests/scaffolding/helpers.ts

```diff
@@ -1,22 +1,143 @@
+import { ApiRx, Keyring } from "@polkadot/api";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { Event } from "@polkadot/types/interfaces";
-import { Codec, ISubmittableResult } from "@polkadot/types/types";
+import { Compact, u128 } from "@polkadot/types";
+import { DispatchError, Event } from "@polkadot/types/interfaces";
+import { AnyNumber, Codec, ISubmittableResult } from "@polkadot/types/types";
 import { u8aToHex, u8aWrapBytes } from "@polkadot/util";
-import { pipe, map } from "rxjs";
+import { mnemonicGenerate } from '@polkadot/util-crypto';
+import { pipe, map, tap, of } from "rxjs";
+import { createKeys } from "./apiConnection";
+import { getAccountInfo, transferFunds } from "./extrinsicHelpers";
 
-export type EventMap = {[key: string]: Event}
-export type Sr25519Signature = {Sr25519: `0x${string}`}
+export const INITIAL_FUNDING = 100000000n;
 
-export function groupEventsByKey() {
+export enum DevAccounts {
```

Comment:
This level of abstraction is probably not needed and makes the code harder to read.

Reply:
I disagree. Some form of constant or enum, I feel, is always better than the proliferation of string literals in code. Consider the following code:
```
createKeys("//Al1ce");
```

This would compile fine, but likely result in a runtime error because the typo in the account name would fail to map to the well-known Substrate dev account `//Alice`.

Now consider the following code:
```
createKeys(DevAccounts.Al1ce);
```

This would fail to compile, alerting the developer of the typo.

Also, the `DevAccounts` enum could be useful in other use cases, say where we want to iterate over dev accounts or something.

Anyway, I don't see a down side to the enum... and since the enum maps exactly to the string values lexically, I don't think it makes the code harder to read at all.

My $0.02.